valkeycompat/adapter.go

```diff
@@ -5333,7 +5333,7 @@ func (c *Compat) FTCreate(ctx context.Context, index string, options *FTCreateOp
 			_cmd = cmds.Incomplete(cmds.FtCreateIndex(_cmd).PayloadField(options.PayloadField))
 		}
 		// [MAXTEXTFIELDS]
-		// FIXME: in go-reids, FTCreateOptions.MaxTextFields should be bool, not int
+		// FIXME: in go-redis, FTCreateOptions.MaxTextFields should be bool, not int
```

Comment:
it's hard to tell when someone is referencing a real strangely named project or when they're just making a typo...